qvisor/src/vmspace/nvidia.rs

```diff
@@ -27,23 +26,44 @@ use crate::xpu::cuda::*;
 
 use cuda_driver_sys::*;
 
+//yiwang
+use std::sync::Arc;
 
 lazy_static! {
     pub static ref NVIDIA_HANDLERS: NvidiaHandlers = NvidiaHandlers::New();
-    pub static ref FUNC_MAP: BTreeMap<ProxyCommand,(XpuLibrary, &'static str)> = BTreeMap::from(
-        [
-            (ProxyCommand::CudaSetDevice, (XpuLibrary::CudaRuntime, "cudaSetDevice")),
-            (ProxyCommand::CudaDeviceSynchronize, (XpuLibrary::CudaRuntime, "cudaDeviceSynchronize")),
-            (ProxyCommand::CudaMalloc, (XpuLibrary::CudaRuntime, "cudaMalloc")),
-            (ProxyCommand::CudaMemcpy, (XpuLibrary::CudaRuntime, "cudaMemcpy")),
-            (ProxyCommand::CudaRegisterFatBinary, (XpuLibrary::CudaDriver, "cuModuleLoadData")),
-            (ProxyCommand::CudaRegisterFunction, (XpuLibrary::CudaDriver, "cuModuleGetFunction")),
-            (ProxyCommand::CudaLaunchKernel, (XpuLibrary::CudaDriver, "cuLaunchKernel")),
-        ]
-    );
+    pub static ref FUNC_MAP: BTreeMap<ProxyCommand, (XpuLibrary, &'static str)> = BTreeMap::from([
```

Comment:
Better to change back. Expect we will have a lot more api defined here. One api per line will be more clear.